src/components/NamePicker.tsx

```diff
@@ -9,10 +9,10 @@ import { makeStyles } from '@material-ui/core/styles';
 
 export default function NamePicker(): JSX.Element {
   const classes = useStyles();
-  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
-  const [anchorEl2, setAnchorEl2] = useState<null | HTMLElement>(null);
-  const [disabled1, setDisabled1] = useState(true);
-  const [disabled2, setDisabled2] = useState(true);
+  const [anchorEl, setAnchorEl] = useState<undefined | HTMLElement>(undefined);
```

Comment:
If you don't pass anything to `useState` it uses `undefined`. Also, if you don't pass anything, TS figures out that the return can be `undefined`. So you can simply further by just 
`const [anchorEl, setAnchorEl] = useState<HTMLElement>();`.

Reply:
Updated and thanks!